sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.1/async_samples/sample_strongly_typing_recognized_form_async.py

```diff
@@ -39,20 +39,20 @@ class Receipt(object):
     https://aka.ms/formrecognizer/receiptfields
     """
 
-    def __init__(self, form):
-        self.receipt_type = form.fields.get("ReceiptType", FormField())
-        self.merchant_name = form.fields.get("MerchantName", FormField())
-        self.merchant_address = form.fields.get("MerchantAddress", FormField())
-        self.merchant_phone_number = form.fields.get("MerchantPhoneNumber", FormField())
-        self.receipt_items = self.convert_to_receipt_item(form.fields.get("Items", FormField()))
-        self.subtotal = form.fields.get("Subtotal", FormField())
-        self.tax = form.fields.get("Tax", FormField())
-        self.tip = form.fields.get("Tip", FormField())
-        self.total = form.fields.get("Total", FormField())
-        self.transaction_date = form.fields.get("TransactionDate", FormField())
-        self.transaction_time = form.fields.get("TransactionTime", FormField())
-
-    def convert_to_receipt_item(self, items):
+    def __init__(self, form: RecognizedForm) -> None:
+        self.receipt_type: FormField  = form.fields.get("ReceiptType", FormField())
+        self.merchant_name: FormField  = form.fields.get("MerchantName", FormField())
+        self.merchant_address: FormField  = form.fields.get("MerchantAddress", FormField())
+        self.merchant_phone_number: FormField  = form.fields.get("MerchantPhoneNumber", FormField())
+        self.receipt_items: list[ReceiptItem]  = self.convert_to_receipt_item(form.fields.get("Items", FormField()))
```

Comment:
generic `list` as a type hint is not supported <3.9. I would just use typing.List